<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersView</class>
 <widget class="QWidget" name="ParametersView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParametersView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lens</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Redshift</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="vDispersionInput">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>50.000000000000000</double>
          </property>
          <property name="value">
           <double>200.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Velocity Dispersion (km/s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="lRedshiftInput">
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Ellipticity (r,theta)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="ellipRInput">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Shear (r,theta)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="ellipThetaInput">
          <property name="toolTip">
           <string>Angle in degrees.</string>
          </property>
          <property name="whatsThis">
           <string>Angle in Degrees.</string>
          </property>
          <property name="maximum">
           <double>359.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="shearRInput">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="shearThetaInput">
          <property name="toolTip">
           <string>Angle in degrees</string>
          </property>
          <property name="whatsThis">
           <string>Angle in Degrees.</string>
          </property>
          <property name="maximum">
           <double>359.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Quasar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Redshift</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="qRedshiftInput">
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="qMassInput">
          <property name="maximum">
           <double>100000000000000004764729344.000000000000000</double>
          </property>
          <property name="value">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Mass (Solar Masses)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="qRadiusInput">
          <property name="maximum">
           <double>10000000000000000000000.000000000000000</double>
          </property>
          <property name="value">
           <double>40.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="qRadiusUnit">
          <item>
           <property name="text">
            <string>R_G</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>uas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Theta_E</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>arcsec</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Root Ray Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rootRayInput">
       <property name="toolTip">
        <string extracomment="The square root of the number of rays you want to send">The square root of the number of rays you want to send</string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="enableMicrolensing">
     <property name="toolTip">
      <string>Selecting this box will cause the ParametersView to generate MicrolensingParameters, rather than a Parameters object for macrolensing</string>
     </property>
     <property name="text">
      <string>Enable Microlensing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="microlensingPane">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Microlensing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Star Generator</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="imfInput">
          <item>
           <property name="text">
            <string>Kroupa_2001</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pooley_2011</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aged_galaxy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Percent Stars</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Source Plane Minor Axis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Image Location (arcsec)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="pcntStarsInput">
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="imgLocInput">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="toolTip">
           <string extracomment="Center of the image you want to simulate microlensing for, relative to the center of the galaxy.">Center of the image you want to simulate microlensing for, relative to the center of the galaxy.</string>
          </property>
          <property name="text">
           <string>(1.2,1.3)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="sPlaneDimUnit">
          <item>
           <property name="text">
            <string>Theta_E</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R_G</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>uas</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="sPlaneDimInput">
          <property name="maximum">
           <double>1000000000000000000.000000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Star Generator Seed</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="imfSeedInput">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="starSeedRandomizer">
          <property name="text">
           <string>Random Seed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customStarGenBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Custom Star Generator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="imfStepsInput">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="toolTip">
           <string>Break points in the broken power law distribution, as well as the extreme values. Should follow the pattern &quot;minimumMass break1 break2 ... maxMass&quot;. Star masses will only be generated between minMass and maxMass, in solar masses.</string>
          </property>
          <property name="whatsThis">
           <string>break points in the broken power law distribution, as well as the extreme values. Should follow the pattern &quot;minimumMass break1 break2 ... maxMass&quot;. Star masses will only be generated between minMass and maxMass, in solar masses.</string>
          </property>
          <property name="text">
           <string>0.08 0.5 1.0 100.0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Powers</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Step Function Breaks</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="imfPowersInput">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="toolTip">
           <string>Exponents for a broken power law mass distribution, space-delimited.</string>
          </property>
          <property name="whatsThis">
           <string>Exponents for a broken power law mass distribution, space-delimited.</string>
          </property>
          <property name="text">
           <string>-0.3 -1.3 -2.3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Aging Conversions</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="imfAgingInput">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="toolTip">
           <string>Threshold values for replacing heavy-mass stars with compact objects. Ex. &quot;0.8:0.6 8.0:1.4&quot; will replace all masses between 0.8 and 8.0 with 0.6, and all masses greater than 8.0 with 1.4</string>
          </property>
          <property name="whatsThis">
           <string>Threshold values for replacing heavy-mass stars with compact objects. Ex. &quot;0.8:0.6 8.0:1.4&quot; will replace all masses between 0.8 and 8.0 with 0.6, and all masses greater than 8.0 with 1.4</string>
          </property>
          <property name="text">
           <string>0.8:0.6 8.0:1.4 30.0:10.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>enableMicrolensing</sender>
   <signal>toggled(bool)</signal>
   <receiver>microlensingPane</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>749</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
